.github/CODEOWNERS

```diff
@@ -1003,6 +1006,9 @@
 # ServiceLabel: %Large Instance %Mgmt
 # ServiceOwners:                                                   @8Gitbrix
 
+# PRLabel: %Monitor
+/sdk/monitor/Azure.ResourceManager.Monitor/                        @thomasp98296 @ritwik8119
```

Comment:
Both of these accounts still lack **public** membership to the Azure and Microsoft orgs.

Reply:
I fixed mine last night. Ritwik just fixed his now. Should take some time to reflect I guess, but thank you for pointing it out.